microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -759,7 +759,7 @@ To manage the collection and use of your search results activity for product imp
 
 1. Select **Settings and more** > **Settings**. 
 
-1. Select **Privacy, search, and services**. 
+1. Select **Privacy and security**. 
```

Comment:
Combine two lines per GUI nav path efficient concise convention:
1. Select **Settings and more** > **Settings** > **Privacy and security**. 
or:
1. Select **Settings and more** > **Settings** > **Search and services**. 

The Suggested Change UI is unavailable here.

Reply:
Correct. this is supposed to be Privacy and security

Reply:
```
To manage the collection and use of your search results activity for product improvement, 
select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** 
> **Privacy and security**, and then turn on or off the setting for 
**Help improve Microsoft products by sending the results from searches on the web**.
```